operator/apis/lifecycle/v1alpha3/common/phases.go

```diff
@@ -0,0 +1,111 @@
+package common
```

Comment:
the stuff in the common package does not seem to have changed - would it make sense to put this one layer up into `apis/lifecycle/common` so that it can be shared across the difference api versions?

Reply:
between v1alpha1 and v1alpha2 has changed, therefore I would keep a separate common for each version, otherwise it may create a mess in the future

Reply:
ok if it is not the same across all versions, then it's a different story